core/test/computed/viewport-meta-test.js

```diff
@@ -52,6 +52,14 @@ describe('ViewportMeta computed artifact', () => {
     assert.equal(parserWarnings[1], 'Invalid values found: {"initial-scale":"microscopic"}');
   });
 
+  it('is not mobile optimized when a viewport contains an initial-scale value lower than 1',
+  async () => {
```

Comment:
nit: if this line is too long I think a `// eslint-disable-next-line max-len` is preferable, or reword to make the test name shorter
```suggestion
  it('is not mobile optimized when a viewport contains an initial-scale value lower than 1', async () => {
```